_includes/sections/instant-messenger.html

```diff
@@ -40,44 +67,199 @@ <h1 id="im" class="anchor"><a href="#im"><i class="fas fa-link anchor-icon"></i>
+    </ul>
+
+    <h3 id="im" class="anchor"><a href="#peer-to-peer"><i class="fas fa-link anchor-icon"></i></a> Peer to Peer</h3>
+
+    <p>These instant messengers connect directly to each other without an authoritative server in between. Peers (clients) usually negotiate connections through the use of some kind of <a href="https://en.wikipedia.org/wiki/Distributed_computing">distributed computing</a> network. Examples of this can include the <a href="https://en.wikipedia.org/wiki/Distributed_hash_table">DHT (Distributed hash table)</a>, or <a href="https://en.wikipedia.org/wiki/Ethereum">Ethereum</a>'s <a href="https://github.com/ethereum/wiki/wiki/Whisper">Whisper</a> protocol.</p>
+
+    <p>Once a peer has found the correct route to its partner, a direct connection to that contact is made. Another approach is proximity based networks, where a connection is established over WiFi or Bluetooth (eg Briar or the <a href="https://www.scuttlebutt.nz">Scuttlebutt</a> social networking protocol).</p>
+
+    <h4>Advantages</h4>
+    <ul>
+        <li>Minimal metadata exposed. Your <a href="https://en.wikipedia.org/wiki/IP_address">IP address</a> and that of the contacts you're communicating with may be visible if you do not use the software in conjunction with an anonymity network like <a href="https://www.torproject.org">Tor</a> or <a href="https://geti2p.net/">I2P</a>. Many countries have some form of mass surveillance and/or metadata retention.</li>
+        <li>Always on E2EE (end-to-end encryption), no ability to disable.</li>
+    </ul>
+
+    <h4>Disadvantages</h4>
+    <ul>
+        <li>Reduced feature set:</li>
+        <ul>
+            <li>No offline messaging (Tox, Jami).</li>
+            <li>Less efficient battery usage on mobile devices because of persistent network connections to the distributed network that allows other peers to find their contacts.</li>
+        </ul>
+    </ul>
+</div>
+
+{%
+  include cardv2.html
+  title="qTox"
+  image="/assets/img/tools/Qtox.png"
+  description="Encrypted instant messaging and video calling software. Uses its <a href \"https://toktok.ltd/spec.html\">own encryption protocol</a> that has not yet been officially audited by cryptographers."
+  labels="warning:<a href=//tox.chat/download.html#warning>Experimental</a>:Encryption has not been audited by professional cryptographers|success:VoIP"
+  website="https://tox.chat"
+  forum=""
+  github="https://github.com/qTox/qTox"
+  windows="https://tox.chat/download.html#oses"
+  mac="https://tox.chat/download.html#oses"
+  linux="https://tox.chat/download.html#oses"
+%}
+
+{%
+  include cardv2.html
+  title="Jami"
+  image="/assets/img/tools/Jami.png"
+  description="Encrypted instant messaging and video calling software. Uses <a href=//en.wikipedia.org/wiki/Transport_Layer_Security>TLS</a> for encryption."
+  labels="success:VoIP"
+  website="https://jami.net/"
+  forum=""
+  gitlab="https://git.ring.cx/savoirfairelinux"
```

Comment:
```suggestion
  gitlab="https://git.jami.net/savoirfairelinux"
```

I think they are moving domains and used to have a warning about updating links.